styles.css

```diff
@@ -288,6 +290,27 @@ footer ul {
   margin-top: 15px;
 }
 
+.modal > ul li {
+  padding: 12px;
+  margin: 6px;
+  border: 1px solid black;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
```

Comment:
- [ ] Hey guys, I must commend your effort in implementing some of the requirements for this project. However, there are some issues I think you should fix. It is very important that your popup follows the design template provided e.g., font, colors, images, text, and margins. Consistency is an essential design requirement. 

- Make the buttons on your popup modal look exactly like the buttons on the Figma template. Ensure that the  button text and icon are properly aligned
- Make sure the text on the right properly aligns with the image as it is on the Figma design and Capitalize the Header text on the popup.
- Reduce the space between the language buttons and the Header text.

> Your Design
![yourFigtem](https://user-images.githubusercontent.com/71140133/232125932-dcee3d47-91b0-4bde-b77c-c0b21aa7313b.PNG)

> Figma Design
![Figtem](https://user-images.githubusercontent.com/71140133/232126012-b3ea6c15-2206-4d52-b238-d8b9026cc5bf.PNG)